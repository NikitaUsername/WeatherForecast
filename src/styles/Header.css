.title {
    color: #757575;
    margin-top: 2rem;
}
.modalForm__input{
    background-color: #dddddd;
    border-radius: 0.65rem;
    margin-bottom: 1rem;
    display: block;
}