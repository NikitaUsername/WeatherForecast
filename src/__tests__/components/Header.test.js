import Header from '../../components/Header';
import React from 'react';
import renderer from 'react-test-renderer'

describe('Header component', () => {
    test('matches the snapshot', () => {
        const tree = renderer
            .create (<Header/>).toJSON();
            expect(tree).toMatchSnapshot();
    });
});