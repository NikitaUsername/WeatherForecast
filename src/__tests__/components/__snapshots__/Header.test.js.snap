// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-lg-4"
    >
      <h2
        id="title"
      >
        Погода здесь
      </h2>
    </div>
    <div
      className="col-lg-4"
    >
      <button
        id="update-button"
        onClick={[Function]}
      >
        Обновить
      </button>
    </div>
  </div>
</div>
`;
